lib/config/definitions.ts

```diff
@@ -461,17 +454,33 @@ const options: RenovateOptions[] = [
     default: false,
   },
   {
-    name: 'trustLevel',
+    name: 'exposeAllEnv',
     description:
-      'Set this to "high" if the bot should trust the repository owners/contents.',
+      'Configure this to true to allow passing of all env variables to package managers.',
```

Comment:
```suggestion
      'Configure this to true to allow passing of all environment variables to package managers.',
```

Maybe spell this out in full?